components/nav.js

```diff
@@ -156,7 +156,8 @@ function Nav({isHomePage, resultObject, errorReport}) {
     <Dropdown simple text='Events' key='Events' style={style}>
       <Dropdown.Menu>
         <Menu.Item as='a' href="/event/upcoming-events" onClick={handleItemClick} style={style}>Upcoming Events</Menu.Item>
-        <Menu.Item as='a' href="/event/women-in-data-conference" onClick={handleItemClick} style={style}>Women in Data Conference</Menu.Item>
+        <Menu.Item as='a' href="/event/women-in-data-conference" onClick={handleItemClick} style={style}>Women in Data Conference 2021</Menu.Item>
```

Comment:
Can we rearrange it? First 2022 and then 2021. I think it will be easier for the user to navigate as most of the visitors want to see the recent conf page